### PR TITLE
[Exporter.Geneva] Remove unnecessary code analysis warning exceptions

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Remove unnecessary code analysis warning exceptions

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackTraceExporter.cs [hunk 1]
```diff
@@ -400,6 +400,23 @@ public void Dispose()
 
     private const int BUFFER_SIZE = 65360; // the maximum ETW payload (inclusive)
 
+    private static readonly string INVALID_SPAN_ID = default(ActivitySpanId).ToHexString();
```
Comment (Kielek) [inline src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackTraceExporter.cs:403]: Probably not part of this PR, but snake case (all caps) is rarely used.

**Review by Kielek — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackTraceExporter.cs (hunk 1 above)
Comment (utpilla) [reply src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackTraceExporter.cs:403]: Yes, this project should use the standard C# naming conventions.
